src/Tools/Source/CompilerGeneratorTools/Source/BoundTreeGenerator/BoundNodeClassWriter.cs

```diff
@@ -308,7 +308,7 @@ private void WriteClassHeader(TreeType node)
                             abstr = "MustInherit ";
                         else if (CanBeSealed(node))
                             abstr = "NotInheritable ";
-                        WriteLine("Friend {1}Partial Class {0}", node.Name, abstr);
+                        WriteLine("Partial {1}Friend Class {0}", node.Name, abstr);
```

Comment:
I believe `Friend` will need to go before `MustInherit ` or `NotInhertiable ` as well.